cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -42,7 +42,7 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	// Run installer with source-path flag (ie. install from this local path)
 	cp := ts.SpawnCmdWithOpts(
 		suite.installerExe,
-		e2e.OptArgs(installationDir(ts)),
+		e2e.OptArgs(installationDir(ts), "-n"),
```

Comment:
Let me know if you think there's a better way of handling this than to just append "-n" to every invocation of "state-installer" in this test suite.